includes/Admin/Admin_Page.php

```diff
@@ -150,12 +152,15 @@ private function get_available_plugins() {
 	 * @since n.e.x.t
 	 */
 	public function render_page() {
-		global $available_plugins, $selected_plugin_basename;
+		global $available_plugins, $selected_plugin_basename, $categories;
 
 		$available_plugins = $this->get_available_plugins();
 
 		$selected_plugin_basename = filter_input( INPUT_GET, 'plugin', FILTER_SANITIZE_FULL_SPECIAL_CHARS );
 
+		$check_categories = new Check_Categories();
+		$categories       = $check_categories->get_categories();
```

Comment:
Static methods don't require an instance.
```suggestion
		$categories = Check_Categories::get_categories();
```